UnityProject/Assets/Scripts/Managers/PlayerList.cs

```diff
@@ -512,6 +512,46 @@ public GameObject TakeLoggedOffPlayerbyUserId(string userId)
 		return null;
 	}
 
+	[Server]
+	public PlayerInfo RemovePlayerbyUserId(string userId, PlayerInfo newPlayer)
```

Comment:
could this instead be a ``TryRemovePlayerByUserId`` and return a bool instead? It can ``out``the ``PlayerInfo``

Reply:
why does it have to be a try?

Reply:
To indicate that there's a possibility it won't succeed e.g. userId is valid. Might be useful for something.